Policies/github-terms-of-service.md

```diff
@@ -36,95 +36,118 @@ Thank you for using GitHub! We're happy you're here. Please read this Terms of S
 - You will promptly [notify GitHub](https://support.github.com/contact) if you become aware of any unauthorized use of, or access to, our Service through your Account, including any unauthorized use of your password or Account.
 
 #### 5. Additional Terms
 In some situations, third parties' terms may apply to your use of GitHub. For example, you may be a member of an organization on GitHub with its own terms or license agreements; you may download an application that integrates with GitHub; or you may use GitHub to authenticate to another service. Please be aware that while these Terms are our full agreement with you, other parties' terms govern their relationships with you.
 
+When you have become, or if you already are:
+- a government User,
+- a User accessing or using any GitHub Service in a government capacity, OR
+- signed up for GitHub Enterprise Cloud,
+
+you are obligated to observe the rest of this section B. Otherwise, you can just skip reading the rest of this section.
+
 If you are a government User or otherwise accessing or using any GitHub Service in a government capacity, this [Government Amendment to GitHub Terms of Service](/articles/amendment-to-github-terms-of-service-applicable-to-u-s-federal-government-users/) applies to you, and you agree to its provisions.
 
 If you have signed up for GitHub Enterprise Cloud, the [Enterprise Cloud Addendum](/articles/github-enterprise-cloud-addendum/) applies to you, and you agree to its provisions.
 
 ### C. Acceptable Use
 **Short version:** *GitHub hosts a wide variety of collaborative projects from all over the world, and that collaboration only works when our users are able to work together in good faith. While using the service, you must follow the terms of this section, which include some restrictions on content you can post, conduct on the service, and other limitations. In short, be excellent to each other.*
 
-Your use of the Website and Service must not violate any applicable laws, including copyright or trademark laws, export control or sanctions laws, or other laws in your jurisdiction. You are responsible for making sure that your use of the Service is in compliance with laws and any applicable regulations.
+Your use of the Website and Service must not violate any applicable laws (including copyright or trademark laws, export control or sanctions laws, or other laws in your jurisdiction), and you are responsible for this compliance.
 
-You agree that you will not under any circumstances violate our [Acceptable Use Policies](https://help.github.com/articles/github-acceptable-use-policies) or [Community Guidelines](https://help.github.com/en/articles/github-community-guidelines).
+You will not under any circumstances violate our [Acceptable Use Policies](https://help.github.com/articles/github-acceptable-use-policies) or [Community Guidelines](https://help.github.com/en/articles/github-community-guidelines).
```

Comment:
`you are not allowed`

Reply:
I'm don't have any legal qualifications, but I would have thought the proposed change is effective for what needs to be communicated. I consider 'you will not under any...'  communicates the same thing as 'you are not allowed to under any...'.

Perhaps you can further explain your comment @KOLANICH?

I also welcome the input of others on this.

Regardless of the above, I'm perfectly happy for it to be changed to 'You are not allowed to under any circumstances..'. A person with contract-drafting experience perhaps could comment on whether we could improve it further by just removing the 'under any circumstances' part.

Reply:
`You will not` sounds more like a prediction.

Reply:
🆗 Now implemented in last commit in patch.